babel.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  presets: [
+    ['@babel/preset-env', { targets: { node: 'current' } }],
+    '@babel/preset-typescript'
+  ]
+}
```

Comment:
Does jest need this to compile individual typescript files, and/or the tests?

Reply:
yep, exactly